extensions/apps/notifications/src/components/pages/notifications-page.tsx

```diff
@@ -149,6 +155,36 @@ const NotificationsPage: React.FC = () => {
       getNavigationUrl: navRoutes => navRoutes['Notifications'],
     });
   };
+  /** Navigate to web3 connector */
+  const handleConnectButtonClick = () => {
+    navigateTo?.({
+      appName: '@akashaorg/app-auth-ewa',
+      getNavigationUrl: (routes: Record<string, string>) => {
+        return `${routes.Connect}?${new URLSearchParams({
+          redirectTo: `${baseRouteName}/`,
+        }).toString()}`;
+      },
+    });
+  };
+
+  if (!isLoggedIn) {
```

Comment:
make sure you are displaying this when the isAuthenticating is also false, otherwise the error loader will show up for a brief moment when is in between